### DISCUSSION TITLE
v0.3.6: static html export!

### DISCUSSION THREAD

**Discussion by oegedijk** [oegedijk/explainerdashboard#133] [General]
### New Features
- Static html export! You can export a static version of the dashboard using the default values
    that you specified in the components or through kwargs with `dashboard.to_html()`.
    - for custom components you need to define your own custom `to_html()` methods, see the documentation.
- A toggle is added to the dashboard header that allows you to download a static export
    of the current live state of the dashboard.
- adds a new toggle and parameter to the ConfusionmatrixComponent to either average
    the percentage over the entire matrix, over the rows or over the columns.
    Set normalize='all', normalize='true', or normalize='pred'. 
- also adds a `save_html(filename)` method to all `ExplainerComponents` and `ExplainerDashboard`
- `ExplainerHub` adds a new parameter `index_to_base_route`: 
    Dispatches Hub to `/base_route/index` instead of the default `/` and `/index`. 
    Useful when the host root is not reserved for the ExplainerHub

<hr /><em>This discussion was created from the release <a href='https://github.com/oegedijk/explainerdashboard/releases/tag/v0.3.6.1'>v0.3.6: static html export!</a>.</em>

**Comment by wllucienne** [comment]
Hi thanks for the nice work. May i check if we could bypass the dummy encoding option as my data as high cardinality?
